doc/api/errors.md

```diff
@@ -2271,6 +2271,17 @@ malconfigured clients, if more than 8KB of HTTP header data is received then
 HTTP parsing will abort without a request or response object being created, and
 an `Error` with this code will be emitted.
 
+<a id="HPE_UNEXPECTED_CONTENT_LENGTH"></a>
+### `HPE_UNEXPECTED_CONTENT_LENGTH`
+
+Server is sending both a Content-Length header and a Transfer-Encoding: chunked
```

Comment:
Not sure if there's a better choice than quotation marks, but something to encase the `Transfer-Encoding: chunked` would be good. Maybe this?
`
```suggestion
Server is sending both a Content-Length header and "Transfer-Encoding: chunked".
```